src/main/java/org/kohsuke/github/GHPullRequest.java

```diff
@@ -536,7 +545,9 @@ public GHPullRequestReviewBuilder createReview() {
      * @return the gh pull request review comment
      * @throws IOException
      *             the io exception
+     * @deprecated use {@link #createReviewComment()}
      */
+    @Deprecated
     public GHPullRequestReviewComment createReviewComment(String body, String sha, String path, int position)
```

Comment:
Replace the implementation of this method with `GHPullRequestReviewCommentBuilder`.